### DISCUSSION TITLE
Default diagnostic severity levels should be reconsidered

### DISCUSSION THREAD

**Discussion by debonte** [microsoft/pylance-release#6084] [Polls]
From Marketplace review:
> Robert Buch
Some useful reports are disabled by default, even on the "strict" type-checking mode. I had to manually set most of these to "warning" or "information". Speaking of, all reports are set to "error" by default, which seems rather unreasonable. Is it really an "error" if you have an unused import? Is the world going to end if I have an unnecessary "isinstance()" call? Such reports should be marked as "warning" or "information".

**Comment by rchiodo** [comment]
I think all of these could be argued either way. 'Strict' means strict so you don't want any reports, even unused imports.

Transferring to a discussion.
